### DISCUSSION TITLE
Mac os - Big sur - how to

### DISCUSSION THREAD

**Discussion by FELS-Zak** [AvaloniaUI/Avalonia#5514] [General]
Hi 
How do I update visual studio for mac so that I can have AvaloniaUI project types ?

**Comment by grokys** [comment]
We don't have an extension for Visual Studio for Mac as it's a completely different product with a different plugin system to the standard VS. However you can still use `dotnet new` from the command-line to create Avalonia projects from templates: https://github.com/AvaloniaUI/avalonia-dotnet-templates

If you want an IDE that runs on Mac and supports Avalonia templates, I'd suggest [JetBrains Rider](https://www.jetbrains.com/rider/).
